ci/rpm/test_daos_node.sh

```diff
@@ -101,17 +101,30 @@ sudo chmod 0755 /tmp/daos_sockets
 sudo chown "$me:$me" /tmp/daos_sockets
 
 FTEST=/usr/lib/daos/TESTING/ftest
-sudo PYTHONPATH="$FTEST/util"                               \
-     $FTEST/config_file_gen.py -n "$HOSTNAME"               \
-                               -a /etc/daos/daos_agent.yml  \
-                               -s /etc/daos/daos_server.yml
+
+python3 -m venv venv
+# shellcheck disable=SC1091
+source venv/bin/activate
+pip install --upgrade pip
+pip install -r $FTEST/requirements-ftest.txt
+
+FTEST=/usr/lib/daos/TESTING/ftest
+sudo PYTHONPATH="$FTEST/util"                        \
+     "${VIRTUAL_ENV}"/bin/python $FTEST/config_file_gen.py -n "$HOSTNAME" \
+        -a /etc/daos/daos_agent.yml -s /etc/daos/daos_server.yml
```

Comment:
This might not work if `config_file_gen.py` has new dependencies that come from the requirements file
```suggestion
sudo PYTHONPATH="$FTEST/util:$PYTHONPATH"                        \
     "${VIRTUAL_ENV}"/bin/python $FTEST/config_file_gen.py -n "$HOSTNAME" \
        -a /etc/daos/daos_agent.yml -s /etc/daos/daos_server.yml
```

Reply:
the activate script doesn't set PYTHONPATH so I think this is OK unless I've misunderstood your point.

Reply:
Then how are the python libs loaded in the env?

Reply:
There's a VIRTUAL_ENV environment variable which I assume python reads from.